collector/adcs.go

```diff
@@ -0,0 +1,246 @@
+//go:build windows
+// +build windows
+
+package collector
+
+import (
+	"errors"
+	"github.com/prometheus-community/windows_exporter/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"strings"
+)
+
+func init() {
+	registerCollector("adcs", ADCSCollectorMethod, "Certification Authority")
+}
+
+type ADCSCollector struct {
```

Comment:
I don't believe this type should be public

Reply:
Ack. My apologies -- this is one of my first projects in GoLang and I was unfamiliar with the case convention for visibility.

Reply:
It's all good, the case sensitivity rather than a private/public keyword trips a lot of people up :+1: